setup.py

```diff
@@ -93,6 +93,7 @@ def alltests():
           'pytz',
           'zope.browser>=1.1',
           'zope.browserpage>=3.11.0',
+          'zope.browserresource',
```

Comment:
So now I've got to ask: don't we need a ZCML directive to register the zope.formlib resource directory?

I'd assumed we had all that done before and you've just added a couple of files into the already-registered directory, but this change raised doubts.

Reply:
Never mind, somehow I've overlooked the change to configure.zcml!